lib/src/unmodifiable_wrappers.dart

```diff
@@ -25,7 +25,7 @@ class NonGrowableListView<E> extends DelegatingList<E>
 
 /// Mixin class that implements a throwing version of all list operations that
 /// change the List's length.
-abstract class NonGrowableListMixin<E> implements List<E> {
+abstract mixin class NonGrowableListMixin<E> implements List<E> {
```

Comment:
Also need to remove comments around `mixin` on line 167 for `UnmodifiableMapMixin`